core/src/main/scala/com/nvidia/spark/rapids/tool/qualification/PluginTypeChecker.scala

```diff
@@ -83,7 +85,13 @@ class PluginTypeChecker extends Logging {
   def getSupportedExprs: Map[String, String] = supportedExprs
 
   private def readOperatorsScore: Map[String, Double] = {
-    val source = Source.fromResource(OPERATORS_SCORE_FILE)
+    var file: String = null
```

Comment:
this would be more scala like written like:

```
val file = platform match {
case "dataproc" => OPERATORS_SCORE_FILE_DATAPROC
      case "emr" => OPERATORS_SCORE_FILE_EMR
      case _ => OPERATORS_SCORE_FILE_ONPREM
    }
```

ideally stay away from vars if possible

Reply:
fixed in latest commit